packages/build/src/log/messages/compatibility.js

```diff
@@ -163,19 +167,34 @@ export const logIncompatiblePlugins = function (logs, pluginsOptions) {
  *
  * @throws Error
  */
-const throwIfOutdatedNextRuntime = function (outdatedPlugins) {
-  const nextOutdatedV4Plugin = outdatedPlugins.find(
-    (plugin) => plugin.package === '@netlify/plugin-nextjs' && semver.satisfies('4.0.0 - 4.26.0'),
-  )
+const throwIfUnsupportedPluginVersion = function (outdatedPlugins) {
+  let packageName
+  let version
+  let latestVersion
+  const nextOutdatedV4Plugin = outdatedPlugins.find((plugin) => {
+    packageName = plugin.pluginPackageJson.name
+    version = plugin.pluginPackageJson.version
+    latestVersion = plugin.latestVersion
+    // https://github.com/npm/node-semver#hyphen-ranges-xyz---abc
+    // semver hyphen range is inclusive 4.0.0 - 4.25.0 is same as >= 4.0.0 || < 4.26.0;
+    return (
+      packageName === '@netlify/plugin-nextjs' &&
+      semver.satisfies(version, '4.0.0 - 4.25.0', { includePrerelease: true })
```

Comment:
FYI for anyone else, this matches expectations of the semver range.
I previously thought the upper number of a dashed semver range was not inclusive.

See [npm semver range documentation](https://docs.npmjs.com/cli/v6/using-npm/semver#hyphen-ranges-xyz---abc)

We also only have a [single release of 4.25.x](https://github.com/netlify/next-runtime/releases?page=2).